app/views/static/ops/log_collection/log_collection.html.haml

```diff
@@ -0,0 +1,47 @@
+%form.form-horizontal#form_div{:name                                 => "angularForm",
+                               'ng-show'                             => "vm.afterGet",
+                               'miq-form'                            => 'true',
+                               'form-changed'                        => 'true',
+                               'model'                               => 'vm.logCollectionModel',
+                               'model-copy'                          => 'vm.modelCopy',
+                               :novalidate                           => 1}
+  = render :partial => "layouts/flash_msg"
+
+  %h3
+    :javascript
+    {{vm.message}}
+
+  .form-group
+    %label.control-label.col-md-2
+      = _("Type")
+    .col-md-8
+      %select{"name" => "log_protocol",
+              "ng-model"                    => "vm.logCollectionModel.log_protocol",
+              "ng-change"                   => "vm.logProtocolChanged()",
+              "ng-options"                  => "choice for choice in vm.selectOptions"}
+
+      :javascript
+        miqInitSelectPicker();
+
+  = render :partial => "layouts/angular-bootstrap/edit_log_depot_settings_angular_bootstrap",
```

Comment:
we should probably convert this first, or make it part of this component 

(no other static partials render enything but `layouts/flash_msg` or `layouts/angular/generic_form_buttons`)